docs/api.md

```diff
@@ -48,6 +49,19 @@ Advanced parameters (optional):
 - `stream`: if `false` the response will be returned as a single response object, rather than a stream of objects
 - `raw`: if `true` no formatting will be applied to the prompt and no context will be returned. You may choose to use the `raw` parameter if you are specifying a full templated prompt in your request to the API, and are managing history yourself.
 
+### JSON mode
+
+Enable JSON mode by setting the `format` parameter to `json`. This will ensure the response is valid JSON:
```

Comment:
```suggestion
Enable JSON mode by setting the `format` parameter to `json` and specifying JSON is required in the prompt. This will ensure the response is valid JSON:
```